src/resource.rs

```diff
@@ -22,7 +22,7 @@ pub use type_def::{TypeDef, TypeDefTag};
 
 /// An error that can occur in computing the signature of a node.
 /// TODO: decide on failure modes
-#[derive(Debug, Clone, Error, PartialEq, Eq)]
+#[derive(Debug, Clone, Error, PartialEq)]
```

Comment:
+1

Reply:
is this a thumbs up?

Reply:
Yes, tho originally I was misremembering that it had to contain arbitrary values - but that was only in a PR where I was including a `ConstTypeError` that could be from checking an arbitrary constant. Actually I'm not sure that this shouldn't be `Eq` after all, it goes back to CustomCheckFail.

Reply:
have reverted